styles.css

```diff
@@ -415,3 +432,243 @@ form {
+    background-size: 65rem;
+    background-position-y: 15px;
+    border-bottom-left-radius: 100px;
+    height: 90vh;
+    width: 100%;
+    margin: 0;
+  }
+
+  .heading,
+  .paragraph-one {
+    padding-left: 16.25rem;
+    margin-left: 0;
+    padding-right: 16.25rem;
+  }
+
+  .heading-one {
+    padding-top: 10.438rem;
+  }
+
+  .head-paragraph,
+  .header-connect {
+    padding-left: 10rem;
+  }
+
+  .lets-connect a {
+    font-size: 16px;
+  }
+
+  .header-socios {
+    padding-left: 10rem;
+  }
+
+  .social-media-links {
+    width: 8.75rem;
+    margin-left: 5.875rem;
+  }
+
+  .social-media-links img {
+    width: 0.875rem;
+    height: 0.938rem;
+  }
+
+  /* desktop version works section */
+  .grid-container {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    background: #e5e5e5;
+    margin-bottom: 20px;
+  }
+
+  #portfolio {
+    margin-left: 20px;
+  }
+
+  .grid-item {
+    display: flex;
+    width: 62.5rem;
+    height: 25rem;
+    justify-content: space-between;
+    padding: 1.25rem;
+  }
+
+  .card-description {
+    display: flex;
+    flex-direction: column;
+    padding: 20px;
+    margin: 10px;
+  }
+
+  .project {
+    width: 50%;
+  }
+
+  .card {
+    display: flex;
+    flex-direction: row;
+    height: 100%;
+  }
+
+  .second-project {
+    order: 2;
+  }
+
+  .fouth-project {
+    order: 2;
+  }
+
+  .project img {
+    width: 100%;
+    height: 100%;
+  }
+
+  .frame {
+    display: flex;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 13px;
+    line-height: 16px;
+    justify-content: space-between;
+    width: 220px;
+  }
+
+  .frame ul {
+    display: flex;
+    list-style-type: disc;
+    justify-content: space-between;
+    width: 180px;
+  }
+
+  .frame li {
+    padding-left: 10px;
+  }
+
+  /* About myself section */
+  #about-myself {
+    display: flex;
+    padding: 0;
+    border-top-right-radius: 100px;
+  }
+
+  #about-myself .heading,
+  .paragraph-one {
+    padding-left: 5.875rem;
+    padding-right: 0;
+  }
+
+  #about-myself .btn {
+    padding-left: 5.875rem;
+    padding-bottom: 15rem;
+  }
+
+  .container h1 {
+    padding-top: 2rem;
+    color: #172b4d;
+  }
+
+  .about-details {
+    width: 45%;
+  }
+
+  .about-details .paragraph-one p {
+    width: 100%;
+  }
+
+  .list-container {
+    width: 55%;
+    padding-top: 2.5rem;
+  }
+
+  .langauges-list {
+    display: flex;
+    flex-direction: row;
+  }
+
+  .sublist {
+    display: flex;
+    flex-direction: column;
+    align-items: baseline;
+    justify-content: center;
+  }
+
+  /* contact form */
+  footer {
+    background-image: url(./images/desktop/contact-form-background-shapes.svg);
+    background-size: 75rem;
+    align-items: center;
+    justify-content: center;
+    margin-top: -10rem;
+    width: 100vw;
+
+    /* height: 646px; */
+  }
+
+  .contact-container {
+    display: flex;
+    flex-direction: column;
+    flex-wrap: wrap;
+    width: 40%;
```

Comment:
- Kindly, center your contact form button and make the description as Figma design.👌👍


| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img src="https://user-images.githubusercontent.com/52010789/206842942-f6796582-5ee4-4984-a643-3d00e83bd6d4.png" width="200">|<img src="https://user-images.githubusercontent.com/52010789/206842943-0831167e-73dc-4b41-949d-c0902abaa484.png" width="200">|


😎️

Reply:
Hi@binhussen,
I aligned the button center to match the figma design.

Thank you.